spring-kafka/src/main/java/org/springframework/kafka/annotation/KafkaBootstrapConfiguration.java

```diff
@@ -53,4 +65,27 @@ public KafkaListenerEndpointRegistry defaultKafkaListenerEndpointRegistry() {
 		return new KafkaListenerEndpointRegistry();
 	}
 
+	static class KStreamsRegistrar implements ImportBeanDefinitionRegistrar, BeanClassLoaderAware {
+
+		private ClassLoader classLoader;
+
+		@Override
+		public void setBeanClassLoader(ClassLoader classLoader) {
+			this.classLoader = classLoader;
+		}
+
+		@Override
+		public void registerBeanDefinitions(AnnotationMetadata importingClassMetadata,
+				BeanDefinitionRegistry registry) {
+			if (ClassUtils.isPresent(org.apache.kafka.streams.kstream.KStreamBuilder.class.getName(),
```

Comment:
Yeah... Really would be better to use just raw name.
I thought it would turn to the string constant, but not.
This is decompiled class:
```
 public void registerBeanDefinitions(AnnotationMetadata importingClassMetadata, BeanDefinitionRegistry registry) {
        if(ClassUtils.isPresent(KStreamBuilder.class.getName(), this.classLoader) && registry.containsBeanDefinition("defaultKafkaStreamsConfig")) {
```
So, it is even worse - the `import` is there to load a class.

I tried to implement `Condition`, but it doesn't see target beans somehow. Only a `@Configuration` on annotation and some common `BPP`s 😢 
Not sure how `@ConditionalOnMissingBean` works in Boot then...